src/TLuaInterpreter.cpp

```diff
@@ -3265,202 +3265,322 @@ int TLuaInterpreter::setBackgroundImage( lua_State *L )
 
 int TLuaInterpreter::setLabelClickCallback( lua_State *L )
 {
-    string luaSendText="";
-    if( ! lua_isstring( L, 1 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if(! pHost) {
+        lua_pushstring( L, tr( "setLabelClickCallback: NULL Host pointer - something is wrong!" )
+                        .toUtf8().constData() );
         lua_error( L );
         return 1;
     }
-    else
-    {
-        luaSendText = lua_tostring( L, 1 );
-    }
-    string luaName="";
-    if( ! lua_isstring( L, 2 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+
+    QString labelName;
+    if( ! lua_isstring( L, 1 ) ) {
+        lua_pushstring( L, tr( "setLabelClickCallback: bad argument #1 type (label name as string expected, got %1!)" )
```

Comment:
If we translate error messages, people in those languages are doing to be at a disavantage because everyone else who doesn't understand their language won't be able to help them. This does not seem like a win for me to put non-English users at a disadvantage like this.

Thoughts @keneanung?